e2e/test/scenarios/collections/collections.cy.spec.js

```diff
@@ -49,7 +49,9 @@ describe("scenarios > collection defaults", () => {
       });
 
       popover().within(() => {
-        cy.findByText(`Collection ${COLLECTIONS_COUNT}`).click();
+        cy.findByText(`Collection ${COLLECTIONS_COUNT}`).click({
+          waitForAnimations: false,
```

Comment:
I left a review on https://github.com/metabase/metabase/pull/29198#discussion_r1135399727, though it seems you've enabled auto-merge. I wonder if we should add a comment explaning why we should do this since it's a hack.